autoload/ember_tools/gf.vim

```diff
@@ -120,26 +120,51 @@ function! ember_tools#gf#Model()
   return ember_tools#ExistingLogicFile('app/models/'.dasherized_name)
 endfunction
 
-function! ember_tools#gf#TemplateComponent()
+"Should be a 'dasherized' name: `foo-bar/bazz`
+function! ember_tools#gf#FindComponent(component_name)
```

Comment:
I try to keep a minimal public interface to make it clear which functions are used elsewhere and which aren't. This function is basically a sort of common helper function, so I would use the `s:` form to keep it script-local, so essentially private. I would name it `s:FindComponent`. Maybe it could be located at the bottom of the file with the other functions it uses, so:

``` vim
function! s:FindComponent(component_name)
function! s:FindComponentLogic(component_name)
function! s:FindComponentTemplate(component_name)
```

Does that make sense?

Reply:
I missed the script-local part. In the process, renamed the function to a more descriptive `s:FindDasherizedComponent`.